js/style.scss

```
@@ -1,143 +1,138 @@
 body {
   background-color: #eeeeee;
-  color: #777777;
+  font-family: sans-serif;
+}
 
-  .container > ul > li {
-    border: transparent 8px solid;
-    border-top: 0px;
-    border-bottom: 0px;
-  }
-  ul {
-    padding: 0;
-    .search {
-      background-color: white;
-      padding: 10px;
-      text-align: center;
-      input {
-        border-radius: 3px;
-        width: 70%;
-        height: 36px;
-        border: 1px solid #eeeeee;
-        &::placeholder {
-          padding: 10px;
-        }
-      }
+.container > ul > li {
+  border: transparent 8px solid;
+  border-top: 0px;
+  border-bottom: 0px;
+}
+ul {
+  padding: 0;
 
-      .fa-search {
-        background-color: #e67d22;
-        padding: 10px;
-        color: #fff;
-        font-size: 20px;
-      }
+  .search {
+    background-color: white;
```

use hex color for the consistency

```suggestion
    background-color: #fff;
```